serde_v8/src/magic/buffer.rs

```diff
@@ -15,22 +16,29 @@ pub enum MagicBuffer {
 }
 
 impl MagicBuffer {
-  pub fn new<'s>(
-    scope: &mut v8::HandleScope<'s>,
-    view: v8::Local<v8::ArrayBufferView>,
-  ) -> Self {
-    Self::try_new(scope, view).unwrap()
+  pub fn empty() -> Self {
+    MagicBuffer::ToV8(Mutex::new(Some(vec![0_u8; 0].into_boxed_slice())))
```

Comment:
untested function

Reply:
And potentially unused, but it's not a new change/func from this PR it was simply moved up since the other funcs were removed and replaced with TryFrom<T>s